flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlCreateTable.java

```diff
@@ -302,6 +326,11 @@ public void unparse(SqlWriter writer, int leftPrec, int rightPrec) {
             writer.newlineAndIndent();
             this.tableLike.unparse(writer, leftPrec, rightPrec);
         }
+
+        if (this.query != null) {
+            writer.newlineAndIndent();
+            this.query.unparse(writer, 0, 0);
```

Comment:
Why not follow the above code to pass `leftPrec` and `rightPrec` to the `unparse` method?